src/pages/spawn/spawnHost/SpawnHostTableActions.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { MockedProvider } from "@apollo/client/testing";
+import { GET_USER } from "gql/queries";
+import { act, fireEvent, render } from "test_utils/test-utils";
+import { CopySSHCommandButton } from "./SpawnHostTableActions";
+
+afterEach(() => {
+  jest.clearAllMocks();
+  jest.clearAllTimers();
+});
+
+const execCommand = jest.fn();
+const user = "bynn.lee";
+const hostUrl = "ec2-54-242-162-135.compute-1.amazonaws.com";
+
+test("CopySSHCommandButton", async () => {
```

Comment:
I think this would make more sense as a description of what this test is actually testing for
```suggestion
describe("CopySSHCommandButton", async () => {
test("Tooltip text should change after clicking on the copy button", async () => {
```